instrumentation/aws-java-sdk-dynamodb-2.15.34/src/main/java/software/amazon/awssdk/services/dynamodb/DefaultDynamoDbClient_Instrumentation.java

```diff
@@ -0,0 +1,137 @@
+        return Weaver.callOriginal();
+    }
+
+    @Trace
+    public ListTablesResponse listTables(ListTablesRequest request) {
+        DynamoDBMetricUtil.metrics(NewRelic.getAgent().getTracedMethod(), "listTables", request.exclusiveStartTableName(), getEndpoint());
+        return Weaver.callOriginal();
+    }
+
+    @Trace
+    public CreateTableResponse createTable(CreateTableRequest request) {
+        DynamoDBMetricUtil.metrics(NewRelic.getAgent().getTracedMethod(), "createTable", request.tableName(), getEndpoint());
+        return Weaver.callOriginal();
+    }
+
+    @Trace
+    public DeleteTableResponse deleteTable(DeleteTableRequest request) {
+        DynamoDBMetricUtil.metrics(NewRelic.getAgent().getTracedMethod(), "deleteTable", request.tableName(), getEndpoint());
+        return Weaver.callOriginal();
+    }
+
+    @Trace
+    public DescribeTableResponse describeTable(DescribeTableRequest request) {
+        DynamoDBMetricUtil.metrics(NewRelic.getAgent().getTracedMethod(), "describeTable", request.tableName(), getEndpoint());
+        return Weaver.callOriginal();
+    }
+
+    @Trace
+    public ScanResponse scan(ScanRequest request) {
+        DynamoDBMetricUtil.metrics(NewRelic.getAgent().getTracedMethod(), "scan", request.tableName(), getEndpoint());
+        return Weaver.callOriginal();
+    }
+
+    @Trace
+    public QueryResponse query(QueryRequest queryRequest) {
+        DynamoDBMetricUtil.metrics(NewRelic.getAgent().getTracedMethod(), "query", queryRequest.tableName(), getEndpoint());
+        return Weaver.callOriginal();
+    }
+
+    @Trace
+    public UpdateItemResponse updateItem(UpdateItemRequest updateItemRequest) {
+        DynamoDBMetricUtil.metrics(NewRelic.getAgent().getTracedMethod(), "updateItem", updateItemRequest.tableName(), getEndpoint());
+        return Weaver.callOriginal();
+    }
+
+    @Trace
+    public UpdateTableResponse updateTable(UpdateTableRequest updateTableRequest) {
+        DynamoDBMetricUtil.metrics(NewRelic.getAgent().getTracedMethod(), "updateTable", updateTableRequest.tableName(), getEndpoint());
+        return Weaver.callOriginal();
+    }
+
+    @Trace
+    public UpdateTimeToLiveResponse updateTimeToLive(UpdateTimeToLiveRequest request) {
+        DynamoDBMetricUtil.metrics(NewRelic.getAgent().getTracedMethod(), "updateTimeToLive", request.tableName(), getEndpoint());
+        return Weaver.callOriginal();
+    }
+
+    @Trace
+    public DescribeLimitsResponse describeLimits(DescribeLimitsRequest describeLimitsRequest) {
+        DynamoDBMetricUtil.metrics(NewRelic.getAgent().getTracedMethod(), "describeLimits", null, getEndpoint());
+        return Weaver.callOriginal();
+    }
+
+    @Trace
+    public TagResourceResponse tagResource(TagResourceRequest tagResourceRequest) {
+        URI endpoint = clientConfiguration != null ? clientConfiguration.option(SdkClientOption.ENDPOINT) : null;
```

Comment:
This var is not being used.
Guess this is leftover from a refactor that created the getEndpoint method.

Reply:
ah yes...thanks!